sample.env

```diff
@@ -1,13 +1,13 @@
 DOCKER_HOST_IP=
-#SDW_GROUP_ID=usdot.jpo.sdw
-#SDW_KAFKA_PORT=9092
```

Comment:
Can we clean up these commented out variables: 
```
#SDW_GROUP_ID=usdot.jpo.sdw
#SDW_KAFKA_PORT=9092
#SDW_DESTINATION_URL=
```
from REAMDME.md, docker-compose.*.yml files as well?

Reply:
SDW_GROUP_ID and SDW_KAFKA_PORT are not referenced in the README or docker-compose files at this time.

I have uncommented SDW_DESTINATION_URL, as this is actually still used by the project.